### DISCUSSION TITLE
Welcome to craft-facebook-conversion Discussions!

### DISCUSSION THREAD

**Discussion by Diewy** [digitalwithyou/craft-facebook-conversion#20] [Announcements]
## 👋 Welcome!

We’re using Discussions as a place to collect all questions and feature requests. Feel free to:
* Ask questions about implementations.
* Share ideas.
* Request new features.

**Comment by mathias-deswarte** [comment]
Hello, is this plugin also used to measure simple conversions? Like certain thank you pages (specific page views)?
